dist/formats/organisation/frontend/schema.json

```diff
@@ -632,6 +632,10 @@
             }
           }
         },
+        "organisation_political": {
```

Comment:
Having a ponder on the name of this and wouldn't it make more sense to be 'political_organisation'?

Reply:
Your point is well taken, and there has been much pondering over the name.  On balance, we've chosen to go with @cbaines suggestion from [here](https://github.com/alphagov/govuk-content-schemas/pull/942#discussion_r356040867).

Reply:
Okay dokay, I'll not re-open a can of worms 😆